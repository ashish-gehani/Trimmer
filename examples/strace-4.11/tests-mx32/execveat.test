#!/bin/sh

# Check execveat syscall decoding.

. "${srcdir=.}/init.sh"

run_prog
run_strace $args
match_grep
run_strace -v $args
match_grep "$LOG" "$srcdir/${ME_%.test}-v.expected"

exit 0
