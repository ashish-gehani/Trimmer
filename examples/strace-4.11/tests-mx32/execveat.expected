execveat\(AT_FDCWD, "execveat\\nfilename", \["execveat\\nfilename", "first", "second"\], \[/\* 2 vars \*/\], AT_SYMLINK_NOFOLLOW\|AT_EMPTY_PATH\) += -1 .*
